packages/storybook/src/stories/ino-card/ino-card.scss

```diff
@@ -63,4 +63,5 @@
 
 ino-card {
   width: 300px;
+  border-radius: 20px;
```

Comment:
This should be moved to the component itself just like in #281.
Otherwise, this breaks the current design

Reply:
Will do ✅